packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx

```diff
@@ -258,6 +258,10 @@ export class SliderBase extends BaseComponent<ISliderProps, ISliderState> implem
       renderedValue: this.state.value
     });
 
+    if (this.props.onBlur) {
```

Comment:
We've had a request to make a small change to the prop. 

1. onBlur should be reserved for the actual onBlur event on the button (mouse release is actually blurring)
2. as we eventually need to change onChange to onChange(event, item) (A breaking change) lets be sure that our new prop uses this new, non deprecated interface


So lets go with onChanged (past tense) with an interface of (ev, value). This will take a few tweaks up in _onMouseDownOrTouchStart to pass the event through.

Reply:
Appreciate the changes @blandine, once these changes are in I'll get this merged and a build will kick off before the next business day.

Reply:
Thanks for the review, I'll change it today (hopefully!).